r-package/paper2020/paper2020.Rmd

```diff
@@ -24,15 +24,18 @@ set.seed(0)
 
 # RESEARCH QUESTIONS AND HYPOTHESES
 
-Transport routing is the process of finding the alternative routes and costs that connect places on a given transport network, and it is a key step in transport accessibility analysis, fleet allocation and transport simulation and planning more broadly [@levinson2020manual]. Researchers and practitioners still frequently face practical challenges when carrying out routing analysis tasks due to the monetary costs of licensed software, limited data availability for data hungry models, and the long computation times required to run multiple routing scenarios, particularly in large and complex multimodal transport networks.
+<!-- edited this because of grammar - costs don't connect places which was grammatically implied in the previous structure -->
+Transport routing is the process of finding the fastest or lowest-cost routes that connect places in a given transport network, and is a key step in transport accessibility analysis, fleet allocation and transport simulation and planning more broadly [@levinson2020manual]. However, researchers and practitioners often face practical challenges when carrying out routing tasks due to the costs of licensed software, limited data availability, and the long computation times required to run multiple routing scenarios, particularly in large and complex multimodal transport networks.
 
-While there is a growing number of open source routing models [@opentripplanneropentripplanner; @lovelace2019stplanr; @padgham2019dodgr], most options available do not efficiently process large transport networks. This paper presents [r5r](https://ipeagit.github.io/r5r/), a new open source R package for routing on multimodal transport networks based on the [Rapid Realistic Routing on Real-world and Reimagined networks (R<sup>5</sup>)](https://github.com/conveyal/r5). R<sup>5</sup> is a powerful next-generation routing engine written in Java and developed at Conveyal [@conway2017evidencebased; @conway2018accounting] to provide an efficient backend for analytic applications - such as accessibility analysis. The r5r package provides a simple and friendly interface to run R<sup>5</sup> locally from within R, which allows users to efficiently calculate travel time matrices or generate multiple route alternatives between origins and destinations using seamless parallel computing.
+<!-- MWC: I'm not sure I agree with this - OTP can handle large networks (and OTP 2.0 actually shares a lot of algorithms with R5), and there are also tools like osrm that can handle really large street routing problems. Is the problem more that the tools that can handle really large networks are inaccessible to researchers? -->
```

Comment:
I agree. A key contribution of `r5r` is making it super easy to do routing analysis

Reply:
in any case, it's still true most available options do 'do not efficiently process large **public** transport networks'.

Reply:
Agreed, I think the contributions are making it easy to do it fast (b/c most easy options are not fast, and most fast options are not easy), and handling large PT networks.